packages/backend/src/services/ServiceRepository.ts

```diff
@@ -894,7 +897,7 @@ export class ServiceRepository
             } else if (factory != null) {
                 serviceInstance = factory();
             } else {
-                throw new Error(
+                throw new MissingConfigError(
```

Comment:
New specific error type for missing services (no license key provided), for checking error type on App.ts